evm_loader/program/src/account_storage/apply.rs

```diff
@@ -230,3 +229,124 @@ impl<'a> ProgramAccountStorage<'a> {
         Ok(())
     }
 }
+
+impl<'a> SyncedAccountStorage for crate::account_storage::ProgramAccountStorage<'a> {
```

Comment:
Can you move this trait implementation to the separate file?

Reply:
Fixed